Example/ConsentViewController_ExampleTests/MessageWebViewControllerSpec.swift

```diff
@@ -22,7 +22,7 @@ class MessageWebViewControllerSpec: QuickSpec, GDPRConsentDelegate, WKNavigation
 
         beforeEach {
             mockConsentDelegate = MockConsentDelegate()
-            messageWebViewController = MessageWebViewController(propertyId: 1, pmId: "1234", consentUUID: "uuid", timeout: 1)
+            messageWebViewController = MessageWebViewController(propertyId: 1, pmId: "1234", consentUUID: "uuid", messageLanguage: MessageLanguage.English, timeout: 1)
```

Comment:
I believe we can use the shorthand notation `.English` instead of `MessageLanguage.English`